cmd/backup-operator/README.md

```diff
@@ -0,0 +1,91 @@
+# etcd backup operator
+
+## Overview
+
+etcd backup operator backups the data of a etcd cluster running on [Kubernetes][Kube] to a remote storage such as AWS [S3][s3].
+
+## Getting Started
+
+You can try out etcd backup operator by running it on Kubernetes and then create a `Backup` Custom Resource which contains the targeting etcd cluster and S3 backup config; the etcd backup operator automatically picks up the `Backup` Custom Resource, retrieves etcd snapshot, and then saves it as a backup to S3.
+>Note: The demo uses the `default` kubernete namespace.
+
+Prerequisites: 
+* access to a Kubernetes environment. See TODO.
+* a running etcd cluster named `example-etcd-cluster`. See [Instruction][etcd_cluster_deploy].
+
+
+Once `example-etcd-cluster` is running, you can create a backup using etcd backup operator. 
+
+First, deploy an etcd backup operator:
+> Note: etcd backup operator also creates Backup CRD when starting.
+
+```sh
+$ kubectl create -f example/etcd-backup-operator/deployment.yaml
+$ kubectl get po
+NAME                                    READY     STATUS    RESTARTS   AGE
+etcd-backup-operator-1102130733-hhgt7   1/1       Running   0          3s
+...
+```
+
+Then, you need to create an Kubernetes secret that contains aws config/credential; etcd operator uses the secret to gain access to S3 in order to save the etcd snapshot.
+
+You need to have aws config and credentials files saved locally:
+```sh
+$ cat $AWS_DIR/credentials
+[default]
+aws_access_key_id = XXX
+aws_secret_access_key = XXX
+
+$ cat $AWS_DIR/config
+[default]
+region = <region>
+```
+
+Create kubernetes secret `aws`:
+
+`$ kubectl create secret generic aws --from-file=$AWS_DIR/credentials --from-file=$AWS_DIR/config`
+
+Modify `example/etcd-backup-operator/backup_cr.yaml` to backup `example-etcd-cluster`:
```

Comment:
Change to:

Assume S3 bucket `my_bucket` is used in this example, create an EtcdBackup yaml:
```
sed 's/<s3-bucket-name>/my_bucket/g' ... > ...
```